internal/helm/helm.go

```diff
@@ -96,7 +96,8 @@ func filterManifests(manifest string) map[string]string {
 	return manifestsToRender
 }
 
-func mergeMaps(a, b map[string]interface{}) map[string]interface{} {
+// Merge maps recursively merges the values of b into a copy of a, preferring the values from b
+func MergeMaps(a, b map[string]interface{}) map[string]interface{} {
```

Comment:
Does this still need to be exported? I don't see any call size outside of `internal/helm`.

Reply:
It doesn't, reverting it now